src/commands/accounts/selectSubscriptions.ts

```diff
@@ -31,14 +32,15 @@ export async function selectSubscriptions(context: IActionContext, options?: Sel
         let subscriptionsShownInPicker: string[] = [];
 
         const subscriptionQuickPickItems: () => Promise<IAzureQuickPickItem<AzureSubscription>[]> = async () => {
-
+            // If there are no tenants selected by default all subscriptions will be shown.
```

Comment:
Not a blocker, but I want to discuss ways we could convey this behavior to users. When you only have a single sub it seems like checking/unchecking tenants does nothing (since the single sub will always be shown).